crates/cli/src/subcommands/subscribe.rs

```diff
@@ -144,7 +143,10 @@ pub async fn exec(config: Config, args: &ArgMatches) -> Result<(), anyhow::Error
 
     // Create the websocket request.
     let mut req = http::Uri::from_parts(uri)?.into_client_request()?;
-    req.headers_mut().insert(header::SEC_WEBSOCKET_PROTOCOL, TEXT_PROTOCOL);
+    req.headers_mut().insert(
+        header::SEC_WEBSOCKET_PROTOCOL,
+        http::HeaderValue::from_static(ws::TEXT_PROTOCOL),
+    );
```

Comment:
why did this need to change?

Reply:
ah I see - this didn't just get moved, it went from having `http::HeaderValue` already baked in, to being a raw string.

(I'm curious why this change had to happen in this PR, but this seems fine as far as my code-owned files are concerned.)

Reply:
Because `client_api_messages` doesn't depend on `http`, so with removing cli's dependency on `client_api`, it now has to be wrapped in cli.

Reply:
okay, then why did we have to remove the CLI's dependency on `client_api` in this PR?

Reply:
It's part of splitting the binaries; spacetimedb-cli shouldn't need to run the `client_api` http server because that's what the spacetimedb-standalone binary does. The follow up also fully removes cli's dependency on `spacetimedb_core`, which cuts out nearly 100 unnecessary dependencies.

Reply:
oh I'm definitely not saying the CLI should be running the server.. but keeping the compile-time dependency doesn't seem like something that had to be changed in this PR.

Either way, not a big deal, it just added some diff and I wasn't sure if I was missing an underlying reason.